src/main/kotlin/no/nav/yrkesskade/meldingmottak/services/SkadeforklaringService.kt

```diff
@@ -69,11 +73,35 @@ class SkadeforklaringService(
         return pdlClient.hentNavn(fodselsnummer)
     }
 
+    private fun opprettDokumenter(vedleggreferanser: List<Vedleggreferanse>, dokumentEierIdentifikator: String): List<Dokument> {
+        return vedleggreferanser.mapNotNull { vedleggreferanse ->
+            opprettDokument(vedleggreferanse, dokumentEierIdentifikator)
+        }.toList()
+    }
+
+    private fun opprettDokument(vedleggreferanse: Vedleggreferanse, dokumentEierIdentifikator: String): Dokument? {
+        val vedlegg: Blob = storageService.hent(vedleggreferanse.id, dokumentEierIdentifikator) ?: return null
+        val filtype = VedleggUtil.utledFiltype(vedlegg.bytes) ?: Filtype.PDF
+
+        return Dokument(
+            brevkode = "",
+            tittel = "Vedlegg",
```

Comment:
Endre tittel til filnavn i stedet for vedlegg.

Reply:
rettet